packages/dds/sequence/src/test/snapshots/OLD/v1/withIntervals.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Seems like something is going wrong with snapshot generation here. I would expect this file to contain the intervals added in this code:

```
        const rand = new Random(Random.engines.mt19937().seed(0));
        const collection1 = sharedString.getIntervalCollection("collection1");
        collection1.add(1, 5, IntervalType.SlideOnRemove, { intervalId: rand.uuid4() });

        const collection2 = sharedString.getIntervalCollection("collection2");
        for (let i = 0; i < sharedString.getLength() - 5; i += 100) {
            collection2.add(i, i + 5, IntervalType.SlideOnRemove, { intervalId: rand.uuid4() });
        }
```

(in generateTestStrings).